doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
 event:
 
     somePromise.then(function(res) {
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+'unhandledRejection':
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
+    }
+
+    var resource = new SomeResource();
+    // no .catch or .then on resource.loaded for at least a turn
+
+To deal with cases like this, which are likely false positives, you can either
```

Comment:
@Fishrock123 the biggest point for unhandled rejection hooks is to un-silence developer errors (but not bug you too much about platform errors). So a developer error like:

```js
JSON.prase('"note it's prase and not parse");
```

Don't get turned into a silent rejection but are somehow obtainable. On the other hand - program errors (like a network stream not opening) might (this is an opinion) be silenced since if you did not `.catch` them you might not care about handling the error (like calling a node method without a callback arg, or ignoring `err`).